src/Hydration.js

```diff
@@ -0,0 +1,40 @@
+import dayjs from 'dayjs'
+import isBetween from 'dayjs/plugin/isBetween'
+dayjs.extend(isBetween)
+
+class Hydration {
+  constructor(userID, filteredUserData) {
+    this.userID = userID
+    this.waterData = filteredUserData[userID].map((waterLogEntry) => {
+      return {
+        date: dayjs(waterLogEntry.date),
+        numOunces: waterLogEntry.numOunces,
+      }
+    })
+  }
+
+  getAvgWater() {
+    return parseFloat((this.waterData.reduce((acc, waterLogEntry) => {
+      return acc + waterLogEntry.numOunces
+    },0)/this.waterData.length).toFixed(2))
+  }
```

Comment:
Generally I would recommend storing the return of an iterator method in a variable because it makes debugging easier if the return comes at the end, especially with longer functions.

ex: 
getAvgWater() {
    const avg =  parseFloat((this.waterData.reduce((acc, waterLogEntry) => {
      return acc + waterLogEntry.numOunces
         },0)/this.waterData.length).toFixed(2))
      return avg
  }